packages/graphql-server/src/resolvers/crd.ts

```diff
@@ -273,8 +274,9 @@ export class Crd<SpecType> {
             return !isEmpty(row.groupName);
           });
         }
+        searchFields = ['name', 'dispalyName', 'description', 'type'];
```

Comment:
I am confused, if want to search resource why not just `name` and `displayName` here?